apps/src/templates/teacherDashboard/sections.js

```diff
@@ -71,6 +77,12 @@ export function renderLoginTypeAndSharingControls(sectionId) {
 }
 
 export function renderSectionTable(sectionId, loginType) {
+  registerReducers({manageStudents});
```

Comment:
I know we talked about adding `registerReducers` to index, but it looks like this file has two other places reducers are registered, so this seemed to make the most sense for now.

Reply:
This seems okay.